src/components/Form/TextField.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+
+import { FormControl, FormLabel, FormErrorMessage } from '@chakra-ui/react';
+import { Field, useField } from 'formik';
+
+import { InputField } from './InputField';
+
+type TextFieldProps = {
+  name: string;
+  label: string;
+  placeholder: string;
+  icon?: ReactNode;
+};
+
+export const TextField: React.FC<TextFieldProps> = ({ label, ...props }) => {
+  console.table(props);
```

Comment:
Unless you need it right now I'd delete it
```suggestion
  console.table(props);
```

Reply:
yep, forgot to remove it. Thanks!